lib/workers/repository/process/limits.ts

```diff
@@ -10,63 +11,98 @@ export async function getPrHourlyRemaining(
 ): Promise<number> {
   if (config.prHourlyLimit) {
     logger.debug('Calculating hourly PRs remaining');
-    const prList = await platform.getPrList();
-    const currentHourStart = moment({
-      hour: moment().hour(),
-    });
-    logger.debug(`currentHourStart=${String(currentHourStart)}`);
     try {
+      const prList = await platform.getPrList();
```

Comment:
As noted below, this returns all PRs, not just ours?

Reply:
Also: will this count ones we created previously this hour? Or will the PR list be cached? Note: we don't want to refetch the entire list just for this purpose though.

Reply:
PR list is cached (checked on github, should also done on other platforms).

We should filter pr list to only include our pr's. Other platforms already use the api for filtering, but github doesn't have that.

Reply:
Hm, I did this by analogy with `getPrHourlyRemaining(...)` implementation. No I see it most probably required fixing too.

Reply:
Should be fixed in seperate pr in platform code. Platform should only return our pr's.